pkg/helmcli/client_test.go

```diff
@@ -0,0 +1,38 @@
+package helmcli
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/gojekfarm/albatross/pkg/helmcli/flags"
+)
+
+type TestSuite struct {
+	suite.Suite
+	c       Client
+	version string
```

Comment:
Do we need the version member as part of the TestSuite struct? Can we have the version var and the check inside the test itself.
Reason: We will add more tests in client and reuse the test suite, so we can make the TestSuite member as slim as possible and let tests define criteria (version in this case)